rfcs/0098_on_device_testing.md

```diff
@@ -0,0 +1,97 @@
+- Feature Name: On-Device Testing in TVM CI
+- Start Date: 2023-01-24
+- RFC PR: [apache/tvm-rfcs#0098](https://github.com/apache/tvm-rfcs/pull/0098)
+- GitHub Issue: [apache/tvm#0000](https://github.com/apache/tvm/issues/0000)
+- Authors: [Mehrdad Hessar](https://github.com/mehrdadh), [David Riazati](https://github.com/driazati) 
+# Summary
+[summary]: #summary
+
+This RFC describes the approach and challenges to add non-merge-blocking hardware testing in TVM CI.
+
+# Motivation
+[motivation]: #motivation
+
+Testing is a major part of any open source project to show its stability to the users and companies who are adopting the project. More than 700 contributors are involved with TVM who works at various companies with different needs/interests in TVM. This means the demand for thorough testing is increasing every day. At the time of writing, TVM tests generally run on the hardware targets when that hardware is available in the cloud (for example, x86 CPU, i386, GPU and AArch64). In addition, TVM has supports hardware targets that are not available in the cloud, such as embedded devices supported by microTVM or the Hexagon DSP. The TVM CI cannot currently test code on those hardware as part of its CI, leaving a gap in testing.
```

Comment:
nit: More than 700 contributors are involved with TVM who work (s/works/work)
nit: In addition, TVM supports (d/has/)